[0m[[0minfo[0m] [0mRunning hw1.initializer [0m
[0m[[0mdebug[0m] [0mWaiting for threads to exit or System.exit to be called.[0m
[0m[[0mdebug[0m] [0m  Classpath:[0m
[0m[[0mdebug[0m] [0m	/home/mithunpaul/fall2017/statNlp_Mihai/hw1/code/target/scala-2.11/classes[0m
[0m[[0mdebug[0m] [0m	/home/mithunpaul/.ivy2/cache/org.scala-lang/scala-library/jars/scala-library-2.11.8.jar[0m
[0m[[0mdebug[0m] [0mWaiting for thread run-main-0 to terminate.[0m
[0m[[0mdebug[0m] [0mInterrupting thread run-main-0[0m
[0m[[0mdebug[0m] [0m	Interrupted run-main-0[0m
[0m[[0mdebug[0m] [0m	Thread run-main-0 exited.[0m
[0m[[0mdebug[0m] [0mInterrupting remaining threads (should be all daemons).[0m
[0m[[0mdebug[0m] [0mSandboxed run complete..[0m
java.lang.RuntimeException: Nonzero exit code: 1
	at scala.sys.package$.error(package.scala:27)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$47$$anonfun$apply$48$$anonfun$apply$49.apply(Defaults.scala:803)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$47$$anonfun$apply$48$$anonfun$apply$49.apply(Defaults.scala:803)
	at scala.Option.foreach(Option.scala:236)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$47$$anonfun$apply$48.apply(Defaults.scala:803)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$47$$anonfun$apply$48.apply(Defaults.scala:801)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[0m[[31merror[0m] [0m(compile:[31mrun[0m) Nonzero exit code: 1[0m
